Documentation/network/servicemesh/gateway-api/installation.rst

```diff
@@ -18,7 +18,14 @@ Prerequisites
     - `ReferenceGrant <https://gateway-api.sigs.k8s.io/api-types/referencegrant/>`_
     - `TLSRoute (experimental) <https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io%2fv1alpha2.TLSRoute/>`_
 
-    .. code-block:: shell-session
+.. admonition:: Warning
+    :class: attention
+
+    Currently, an issue in Gateway API requires Cilium having the experimental CRDs to be installed in order to use Gateway API. The issue is being tracked `here <https://github.com/kubernetes-sigs/gateway-api/issues/3080>`__.
```

Comment:
“Currently” is only guaranteed to be accurate at the time of writing. Let's remove it.

```suggestion
    An issue in Gateway API requires Cilium having the experimental CRDs to be installed in order to use Gateway API. The issue is being tracked `here <https://github.com/kubernetes-sigs/gateway-api/issues/3080>`__.
```